_events/2014-06-16-scaladays.md

```diff
@@ -6,5 +6,5 @@ location: Berlin, Germany
 description: ""
 start: 16 June 2014
 end: 18 June 2014
-link-out: https://scaladays.org/assets/archive/berlin2014/index.html
+link-out: https://scaladays.org/
```

Comment:
Did we lose historical Scala Days sites? If so could we bring them back?

Reply:
hmm, not sure who to ask, maybe @MaureenElsberry ?

I think we should merge the PR regardless to fix CI, but I'd be happy to revert it again later if the archives were resurrected.